components/mpas-ocean/src/Registry.xml

```diff
@@ -2702,11 +2702,9 @@
 		/>
 		<var name="vertGMBolusVelocityTop" type="real" dimensions="nVertLevelsP1 nCells Time" units="m s^{-1}"
 			 description="vertical tracer-transport velocity defined at center (horizontally) and top (vertically) of cell.  This is not the vertical ALE transport, but is Eulerian (fixed-frame) in the vertical, and computed from the continuity equation from the horizontal GM Bolus velocity."
-			 packages="gm"
 		/>
```

Comment:
@mark-petersen why are these removed from the packages?  I had reworked the ocn_diagnostics routine to allow this to be done.  Did I forget changes further upstream the call tree?  I think we should move to those being packages so we don't have to carry massive arrays of zeros for high res runs.